src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Collections/Main.java

```diff
@@ -0,0 +1,70 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Collections;
+
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+        // ITEMS CREATED
+        Item itemNrOne = new Item(3000L, "Azura Dress", Type.CLOTHES);//call enum Enum.Value
+        Item itemNrTwo = new Item(3000L, "Swimsuit", Type.CLOTHES);
+        Item itemNrThree = new Item(3000L, "Leather Jacket", Type.CLOTHES);
+        Item itemNrFour = new Item(3000L, "Fur Coat", Type.CLOTHES);
+        Item itemNrFive = new Item(3000L, "VR Station", Type.GADGETS);
+        Item itemNrSix = new Item(3000L, "Kindle 10th Generation", Type.GADGETS);
+        Item itemNrSeven = new Item(4000L, "LEGO", Type.TOYS);
+        Item itemNrEight = new Item(4000L, "JENGA", Type.TOYS);
+
+        // LIST OF ITEMS CREATED
+        // ----------LIST OF ITEMS BOUGHT FOR ONE USER
+        List<Item> firstItemList = new ArrayList<Item>();
+        firstItemList.addAll(Arrays.asList(itemNrEight, itemNrOne, itemNrSix));
+        System.out.println(firstItemList);
+
+        // ----------LIST OF ITEMS BOUGHT FOR SECOND USER
+        List<Item> secondItemList = new ArrayList<Item>();
+        secondItemList.addAll(Arrays.asList(itemNrFive));
+        System.out.println(secondItemList);
+
+        // ----------LIST OF ITEMS BOUGHT FOR A THIRD USER
+        List<Item> thirdItemList = new ArrayList<Item>();
+        thirdItemList.addAll(Arrays.asList(itemNrThree, itemNrFive));
+        System.out.println(thirdItemList);
+
+        // ----------LIST OF ITEMS BOUGHT FOR A FOURTH USER
+        List<Item> fourthItemList = new ArrayList<Item>();
+        fourthItemList.addAll(Arrays.asList(itemNrOne, itemNrSix, itemNrTwo));
+        System.out.println(fourthItemList);
+
+        // ----------LIST OF ITEMS BOUGHT FOR A FIFTH USER
+        List<Item> fifthItemList = new ArrayList<Item>();
+        fifthItemList.addAll(Arrays.asList(itemNrFour));
+        System.out.println(fifthItemList);
+
+        // ATTACH ADDRESS FOR A USER. USER CREATED
+        //-----------USER1
+        Map userOneAddress = new HashMap<String, Address>(); //tip de date pentru Keys si valorile care le va primi la crearea adresei din clasa care ne trebuie
+        userOneAddress.put("HOME ADDRESS", new Address("MD", "Chisinau", "Rozelor 5A"));
```

Comment:
You no need to initialize new HashMap every time you create your address, initialize it into default constructor, after that call default constructor into the overloaded constructor